packages/devtools_app/test/shared/widget_rebuild_counts_test.dart

```diff
@@ -118,6 +118,3 @@ const event2 =
     '{"startTime":21386348,"frameNumber":2,"events":[7,1,9,11,10,11,11,11,12,11]}';
 const event3 =
     '{"startTime":22385849,"frameNumber":3,"events":[7,1,9,11,10,11,11,11,12,11]}';
-
-const locationJson =
```

Comment:
please leave this for now, as this feature is under construction and this locationJson helper may be used when completing the test coverage for this feature

Reply:
Done